data/domain/rift.tsx

```diff
@@ -86,7 +86,7 @@ export class SkillMastery extends RiftBonus {
             case level < 500: return 4;
             case level < 750: return 5;
             case level < 1000: return 6;
-            case level > 1000: return 7;
+            case level >= 1000: return 7;
```

Comment:
I wonder if all previous levels don't need the same treatment, i.e do you unlock the bonus on 250 or 251 etc.

Reply:
I don't think so, for example, bonus 6 is unlocked when reaching 750

if level = 750, then (level < 750) would be false so it'd go to the next case which is (level < 1000) which would result in yes, thus returning 6